apis/externalsecrets/v1beta1/externalsecret_types.go

```diff
@@ -490,7 +489,6 @@ type ExternalSecretStatus struct {
 }
 
 // +kubebuilder:object:root=true
-// +kubebuilder:storageversion
```

Comment:
Still leaving v1beta1 for allowing a smooth (non-breaking)) transition path for users